src/sdl2/audio.rs

```diff
@@ -65,18 +65,19 @@ use rwops::RWops;
 
 use sys::audio as ll;
 
-impl AudioSubsystem {
+impl<'a> AudioSubsystem {
```

Comment:
I'm not sure you have to add this lifetime globally, I think you can add them manually to every `fn` that use it and only them. From a quick glance it looks like it's only used from `&'a str` which aren't used that much here, so if the compiler allows it please move this lifetime from here and add it to individual functions instead.